src/applications/check-in/api/local-mock-api/e2e/ApiInitializer.js

```diff
@@ -484,14 +484,14 @@ class ApiInitializer {
 
   initializeBtsssPost = {
     withSuccess: () => {
-      cy.intercept('POST', `/check_in/v0/travel_claims/`, req => {
+      cy.intercept(`/check_in/v0/travel_claims/`, { times: 1 }, req => {
```

Comment:
I think this change breaks other tests, I think you will need to make this a var that gets passed through

Reply:
tests is `multiple-facility.cypress.spec.js` are failing